openlane_run/action.yml

```diff
@@ -0,0 +1,31 @@
+name: Run design with latest OpenLane release
+
+inputs:
+  design_name:
+    description: Name of design to run
+    required: true
+  pdk_root:
+    description: PDK root path
+    required: true
+  ol_tag:
+    description: OpenLane tag
+    default: latest
```

Comment:
should the default be the version stored in `openlane-stable-tag` ?

Reply:
I think `latest` is easier to set as a default. The goal is that the user would have on their repo two workflows one that uses `openlane_run` with `latest` and the other with the stable tag.